test/parallel/test-dotenv.js

```diff
@@ -76,7 +76,7 @@ assert.strictEqual(process.env.EDGE_CASE_INLINE_COMMENTS, 'VALUE1');
 assert.strictEqual(process.env.MULTI_DOUBLE_QUOTED, 'THIS\nIS\nA\nMULTILINE\nSTRING');
 assert.strictEqual(process.env.MULTI_SINGLE_QUOTED, 'THIS\nIS\nA\nMULTILINE\nSTRING');
 assert.strictEqual(process.env.MULTI_BACKTICKED, 'THIS\nIS\nA\n"MULTILINE\'S"\nSTRING');
-assert.strictEqual(process.env.MULTI_NOT_VALID_QUOTE, '"');
+assert.strictEqual(process.env.MULTI_NOT_VALID_QUOTE, '');
```

Comment:
does this cover all the added code?

Reply:
The `MULTI_NOT_VALID_QUOTE` test makes sure that if there's a line starting with a double quote that doesn't close, we still count it as one single value.
```
MULTI_NOT_VALID_QUOTE="
```
IMO we should keep this test to make sure we handle these situations right

Reply:
Dotenv interprets this as a `single quote "` too not an empty value

Reply:
yes this preserves the previous code

Reply:
I must have done this wrong I will update it today